src/main/kotlin/kr/io/snuhbmilab/carediaryserverv2/external/sqs/listener/DiaryAnalysisResponseListener.kt

```
@@ -0,0 +1,31 @@
+
+    @SqsListener(
+        value = [$$"${aws.sqs.model-output-queue-url}"],
+        maxConcurrentMessages = "10",
+        maxMessagesPerPoll = "10"
+    )
+    fun onMessage(response: DiaryAnalysisResponse) {
+        try {
+            logger.info { "SQS 분석 응답 수신 - diaryId: ${response.diaryId}, PIE-items: ${response.items.size}개" }
+            diaryAnalysisResultService.saveAnalysisResult(response)
+            logger.info { "분석 결과 저장 완료 - diaryId: ${response.diaryId}" }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)